packages/web3-eth-ens/src/ressources/ABI/Resolver.js

```diff
@@ -1,23 +1,158 @@
 "use strict";
 
+// This ABI taken from @ensdomains/resolver@0.2.4
+// @ensdomains/resolver/build/contracts/Resolver.json
```

Comment:
I once thought about using those ABI's from the ENS packages itself.. we could/should do that and add the additional required ABI items for backward compatibility with a clear deprecation guide/timeline. :-)

Reply:
Agree, that's better. Fixed in efd5cbf